crates/libzkp/src/tasks/batch.rs

```diff
@@ -44,14 +44,14 @@ pub enum BatchHeaderV {
     ///
     /// Since the codec essentially is unchanged for the above STF versions, we do not define new
     /// variants, instead re-using the [`BatchHeaderV7`] variant.
-    V7_V8_V9(BatchHeaderV7),
+    V7_V8_V9_V10(BatchHeaderV7),
 }
 
 impl core::fmt::Display for BatchHeaderV {
     fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
         match self {
             BatchHeaderV::V6(_) => write!(f, "V6"),
-            BatchHeaderV::V7_V8_V9(_) => write!(f, "V7_V8_V9"),
+            BatchHeaderV::V7_V8_V9_V10(_) => write!(f, "V7_V8_V9_V10"),
```

Comment:
This looks incorrec, we'll continue to use batch header v9, at least that was my understanding. And that seems to be the intention in https://github.com/scroll-tech/zkvm-prover/pull/236